Model/Repositories/FlightRepository.cs

```diff
@@ -0,0 +1,25 @@
+﻿using AirportTicketBooking.Model.Classes;
+using AirportTicketBooking.Utils;
+
+namespace AirportTicketBooking.Model.Repositories;
+
+public static class FlightRepository
+{
+    private static List<Flight> _flights = [];
+
+    public static List<Flight> GetAllFlights()
+    {
+        ReadFlightsFromFile();
```

Comment:
Can we Initiate data in prior? I mean this method looks like it has 2 things to do it will load data then it will return them
and each time we call this method it will read from the file